_posts/2024-11-27-driving-community-contributions.md

```diff
@@ -0,0 +1,41 @@
+meta_keywords: 
+meta_description: 
+---
+
+When Amazon Web Services (AWS) launched the OpenSearch Project, it sought to inspire a community of contributors to help drive innovation for a strategically important open-source project. Working toward that goal called for new domain knowledge and a fresh look at company culture to help build trust across internal teams and, more importantly, with external contributors and organizations. 
+
+In [this keynote address](https://www.youtube.com/watch?v=9gwTMW901ew&t=4142s), I shared the journey OpenSearch has embarked on with the invaluable support of our community, highlighting our efforts to empower and collaboratively shape the project's future.
+
+<img src="/assets/media/blog-images/2024-11-27-driving-community-contributions/LF_Member_Summit_Keynote_journey.png" alt="OpenSearch Project Linux Foundation infographic" class="center"/>{:style="width: 100%; max-width: 800px; height: auto; text-align: center"}
+
+For those unfamiliar with OpenSearch, it is a community-driven, open-source platform for search, analytics, and vector databases. It includes integrated tools for observability, security, visualization, and AI-powered applications, all available under the Apache 2.0 license. OpenSearch is now a project under the Linux Foundation. At the Open Source Summit Europe in Vienna two months ago, we launched the [OpenSearch Software Foundation](https://foundation.opensearch.org/) to establish open governance for the project. Since then, we've formed a governing board and a technical steering committee, which I have the honor of leading, to guide the project's technical direction.
+
+OpenSearch originated as a fork of Elasticsearch in early 2021 after Elastic changed its licensing to a more restrictive source-available model. Users still desired an open-source alternative, prompting AWS to release a fork. This phase of our journey was both exciting and challenging. The fork involved substantial effort: 650 pull requests, around 56,000 files, and over 4.5 million lines of code to modify. By July, we delivered a feature-complete, stable version 1.0. At that time, OpenSearch was one of the few fully open-source search and analytics engines available. Our primary focus was to launch a stable, interoperable 1.x line that met our community's needs.
+
+With a stable product in place, 2022 was dedicated to listening and learning—--a lot of learning. As we invited and encouraged a community of users and contributors, both the OpenSearch team and our sponsors at AWS spent the year understanding how to enable a community independently. This might seem surprising, but AWS had not previously led an open-source project of OpenSearch's scope and trajectory. We had to learn the best ways to build trust with internal AWS teams and, more importantly, with external contributors and organizations. To support this, we engaged through various communication channels—forums, blog posts, and numerous requests for comments.
```

Comment:
```suggestion
With a stable product in place, 2022 was dedicated to listening and learning---a lot of learning. As we invited and encouraged a community of users and contributors, both the OpenSearch team and our sponsors at AWS spent the year understanding how to enable a community independently. This might seem surprising, but AWS had not previously led an open-source project of OpenSearch's scope and trajectory. We had to learn the best ways to build trust with internal AWS teams and, more importantly, with external contributors and organizations. To support this, we engaged through various communication channels---forums, blog posts, and numerous requests for comments.
```